src/main.cpp

```diff
@@ -50,7 +50,7 @@ void writeStats(std::fstream &file, std::vector<std::pair<std::string, int>> &st
 
 int main() {
     std::string filename = "../player.csv";
-    std::fstream file("../player.csv" , file.in | file.out);
+    std::fstream file("../player.csv");
```

Comment:
It is wrong than in the standard setup, the player.csv lays in the same directory as the executable doppelkopf, but this path says it has to be one folder higher.

Reply:
will probably be solved by doing an out of source build with cmake